base/changes.txt

```diff
@@ -6,9 +6,16 @@ completeness or accuracy and it contains some references to files that
 are not part of the distribution.
 ================================================================================
 
+2021-12-28  Frank Mittelbach  <Frank.Mittelbach@latex-project.org>
+
+	* ltexpl.dtx (section{Document-level command names for expl3 functions}):
+	Added document.level names for \exp_args:Nc and the like
```

Comment:
typo: document.level -> document-level (sorry it's a late comment)

Reply:
handles in https://github.com/latex3/latex2e/commit/9355fdc9f40afd0aa52f993ac4f218fa444ce964. thanks